src/resources/2014/ConspiracyIsland.ts

```diff
@@ -0,0 +1,296 @@
+    20,
+    "questESpJunglePun",
+    11,
+    "junglePuns",
+    true,
+    false,
+    false,
+    $location`The Deep Dark Jungle`,
+    $item`encrypted micro-cassette recorder`,
+    "mys"
+  ),
+  new QuestData(
+    "OutOfOrder",
+    30,
+    "questESpOutOfOrder",
+    99,
+    "",
+    true,
+    false,
+    false,
+    $location`The Deep Dark Jungle`,
+    $item`GPS-tracking wristwatch`,
+    "10 Initiative"
+  ),
+  new QuestData(
+    "Smokes",
+    30,
+    "questESpSmokes",
+    10,
+    "",
+    false,
+    false,
+    false,
+    $location`The Deep Dark Jungle`,
+    $item`none`,
+    "10 item, 10 pickpocket chance"
+  ),
+] as const;
+
+export function available(): boolean {
+  return have() || get("_spookyAirportToday");
+}
+
+const BLANK_QUEST = new QuestData(
+  "",
+  0,
+  "",
+  0,
+  "",
+  false,
+  false,
+  false,
+  $location`none`,
+  $item`none`,
+  ""
+);
+
+export function completedOneTimeQuests(): boolean {
+  return quests
+    .filter((q) => q.reward === 30)
+    .every((q) => get(q.questStateProperty) === "finished");
+}
+
+export function hasQuest(): boolean {
+  return quests.some((q) => q.currentQuest());
+}
+
+export function activeQuest(): QuestData {
+  return quests.find((q) => q.currentQuest()) || BLANK_QUEST;
+}
+
+export function acceptQuest(): void {
+  if (!hasQuest() && canInteract() && available()) {
+    visitUrl(radio);
+    runChoice(1);
+  }
+}
+
+export function questComplete(): boolean {
+  const quest = activeQuest();
+  return (
+    quest !== BLANK_QUEST &&
+    get(quest.questStateProperty) ===
+      (quest.requiredItem === $item`none` ? "step1" : "step2")
+  );
+}
+
+export function hasActiveQuest(): boolean {
+  return hasQuest() && !questComplete();
+}
+
+export function turnInQuest(): void {
+  if (questComplete()) {
+    visitUrl(radio);
+    runChoice(1);
+  }
+}
+
+export const protocols = new Map<string, [Monster, Item, number]>([
+  ["Juan", [$monster`government scientist`, $item`none`, 1]],
+  ["3.14", [$monster`government scientist`, $item`battery-powered drill`, 2]],
+  ["D-R.00G", [$monster`lab monkey`, $item`none`, 3]],
+  ["B.A.R.F.", [$monster`lab monkey`, $item`monkey barf`, 4]],
+  ["VO-5", [$monster`creepy little girl`, $item`none`, 5]],
+  ["R-XPN", [$monster`creepy little girl`, $item`cuddly teddy bear`, 6]],
+  ["1862", [$monster`super-sized Cola Wars soldier`, $item`none`, 7]],
+  [
+    "1912",
+    [$monster`super-sized Cola Wars soldier`, $item`khaki duffel bag`, 8],
+  ],
+  ["0-0Z-E", [$monster`none`, $item`airborne mutagen`, 9]],
+  ["Ω", [$monster`none`, $item`none`, 10]],
+]);
+
+export function activateProtocol(protocol: number | string) {
+  visitUrl(omegapanel);
+  if (typeof protocol === "number") {
+    runChoice(protocol);
+  } else {
+    for (const button of protocols) {
+      const name = button[0];
+      if (name === protocol) {
+        runChoice(button[1][2]);
+      }
+    }
+  }
+}
+
+export function activateOmega(): void {
```

Comment:
Can we get this to return a boolean of whether omega was successfully activated?

Reply:
Agreed